src/daemon/XorgDisplayServer.cpp

```diff
@@ -261,15 +261,18 @@ namespace SDDM {
         // create display setup script process
         QProcess *displayScript = new QProcess();
 
+        const QString xcursorTheme = mainConfig.Theme.CursorTheme.get(),
+                      xcursorSize = mainConfig.Theme.CursorSize.get();
+
         // set process environment
         QProcessEnvironment env;
         env.insert(QStringLiteral("DISPLAY"), m_display);
         env.insert(QStringLiteral("HOME"), QStringLiteral("/"));
         env.insert(QStringLiteral("PATH"), mainConfig.Users.DefaultPath.get());
         env.insert(QStringLiteral("XAUTHORITY"), m_xauth.authPath());
         env.insert(QStringLiteral("SHELL"), QStringLiteral("/bin/sh"));
-        env.insert(QStringLiteral("XCURSOR_THEME"), mainConfig.Theme.CursorTheme.get());
-        QString xcursorSize = mainConfig.Theme.CursorSize.get();
```

Comment:
Isn't this two solutions to the same problem?

Reply:
IIRC, both are needed. The environment variable one for `xsetroot` and other users of libXcursor, and the resource for Qt applications and maybe other frameworks.

Reply:
Whilst that's absolutely true on the desktop, we know what toolkit the greeter uses. 
Anything outside the greeter seems like the desktop's problem

Reply:
Technically works fine.

Qt 6's own platform code (which isn't used anywhere for cursor stuff?) only looks at the environment variables though, so let's just keep it around for now? Also makes it less likely to break something...